guild.go

```diff
@@ -315,25 +249,16 @@ func (c *Client) ListGuildMembers(guild objects.Snowflake, params *ListGuildMemb
 
 	u.RawQuery = v.Encode()
 
-	res, err := c.request(&request{
-		method:      http.MethodGet,
-		path:        u.String(),
-		contentType: JsonContentType,
-	})
-	if err != nil {
-		return nil, err
-	}
+	members := []*objects.GuildMember{}
+	err = NewRequest().
+		Method(http.MethodGet).
+		Path(u.String()).
+		ContentType(JsonContentType).
+		Expect(http.StatusOK).
+		Bind(members).
```

Comment:
We need to pass a pointer to the slice here, otherwise it will automatically return an error.